src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -19,7 +19,16 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="Microsoft.DotNet.Build.CloudTestTasks\*.dll" target="lib\net45" />
+    <file src="Microsoft.DotNet.Build.Tasks.net45\*.dll" target="lib\net45" />
+    <file src="Microsoft.DotNet.Build.CloudTestTasks\Microsoft.Azure.KeyVault.Core.dll" target="lib\net45" />
```

Comment:
There are three dependencies that exist in both the cloudTestTasks and the net45 task library, they are Newtonsoft.Json, System.Reflection.Metadata and System.Collections.Immutable. This causes no issues for now as they both projects have the same version of all three assemblies. When this stops being true, we can start dividing each project into a different folder within net45.